alacritty_terminal/src/config/bell.rs

```diff
@@ -0,0 +1,122 @@
+use std::time::Duration;
+
+use log::error;
+use serde::{Deserialize, Deserializer};
+use serde_yaml::Value;
+
+use crate::config::{failure_default, Program, LOG_TARGET_CONFIG};
+use crate::term::color::Rgb;
+
+const DEFAULT_BELL_COLOR: Rgb = Rgb { r: 255, g: 255, b: 255 };
+
+#[serde(default)]
+#[derive(Deserialize, Clone, Debug, PartialEq, Eq)]
+pub struct BellConfig {
+    /// Visual bell animation function.
+    #[serde(deserialize_with = "failure_default")]
+    pub animation: BellAnimation,
+
+    /// Visual bell duration in milliseconds.
+    #[serde(deserialize_with = "failure_default")]
+    duration: u16,
+
+    /// Visual bell flash color.
+    #[serde(deserialize_with = "deserialize_bell_color")]
+    pub color: Rgb,
+
+    /// Command to run on bell.
+    #[serde(deserialize_with = "deserialize_bell_command")]
+    pub command: Option<Program>,
+}
+
+impl Default for BellConfig {
+    fn default() -> Self {
+        Self {
+            animation: Default::default(),
+            duration: Default::default(),
+            color: DEFAULT_BELL_COLOR,
+            command: Default::default(),
```

Comment:
```suggestion
            command: Default::default(),
            color: DEFAULT_BELL_COLOR,
```

Doesn't matter at all, but this would look so much better. :)